lib/plugin.js

```diff
@@ -1,44 +1,63 @@
 class ReCaptcha {
   constructor ({ hideBadge, siteKey }) {
     if (!siteKey) {
-      throw 'ReCaptcha error: No key provided'
+      throw new Error('ReCaptcha error: No key provided')
     }
 
+    this._ready = false
+
     this.siteKey = siteKey
+    this.hideBadge = hideBadge
+  }
+
+  init () {
```

Comment:
Maybe 
```suggestion
  async init () {
```

Or do you try to avoid that? :)

Reply:
@manniL Do you have an idea how to make it the better way?

Reply:
This makes extra code after transpilation also extra wrapper promise. Anyway we need internal `new Promise`

Reply:
@pi0 Wondering why we'd need a `new Promise` inside? return values from async functions are always a Promise I thought 🤔

Reply:
@pi0 Ah, because of line 32. nvm